website/src/app.pcss

```diff
@@ -7,6 +7,14 @@
 @tailwind components;
 @tailwind utilities;
 
+html.dark {
+	@apply bg-white min-h-full;
```

Comment:
This part is to avoid pages with limited content from show a white area that is unoccupied by the html element.